lib/graphql/static_validation/rules/variable_default_values_are_correctly_typed.rb

```diff
@@ -23,8 +23,8 @@ def validate_default_value(node, context)
           else
             begin
               valid = context.valid_literal?(value, type)
-            rescue GraphQL::CoercionError => err
-              error_message = err.message
+            rescue GraphQL::CoercionError, GraphQL::LiteralValidationError => err
+              error_message = err.message if err.is_a? GraphQL::CoercionError
```

Comment:
It looks like one kind of error should be handled one way, while another kind of error should be handled another way, is that right? Why not use different rescue clauses for this, for example 

```
begin 
  # ...
rescue GraphQL::CoercionError => err 
  # as before 
rescue GraphQL::LiteralValidationError 
  # do nothing, but don't let it propagate. 
end 
```

?

Reply:
fixed.